generator/generator_config.textproto

```diff
@@ -145,6 +145,23 @@ service {
 }
 
 # Bigtable
+service {
+  service_proto_path: "google/bigtable/v2/bigtable.proto"
+  product_path: "google/cloud/bigtable"
+  initial_copyright_year: "2022"
+  omit_client: true
+  omit_connection: true
+  omitted_rpcs: [
+   "CheckAndMutateRow",
```

Comment:
Did you mean to omit all the RPCs?

Reply:
Yes you did, it was part of your plan in the design doc.

Reply:
Oops. I did mean to omit all RPCs, but it was a dumb plan. (Originally I thought I would be generating a Connection at this step too).

We should just generate the stub code (that we can) now. Also, I updated `codecov.yml` to ignore the stub code that will stay generated.

PTAL